tests/service/StructureTests.fs

```diff
@@ -60,16 +60,15 @@ let (=>) (source: string) (expectedRanges: (Range * Range) list) =
         reraise()
 
 [<Test>]
-let ``empty file``() = "" => [ (1, 0, 2, 0), (1, 0, 2, 0) ]
+let ``empty file``() = "" => []
 
 [<Test>]
 let ``nested module``() =
     """
 module MyModule =
     ()
 """
-    => [ (1, 0, 4, 0), (1, 0, 4, 0)
```

Comment:
Looking at this again, why are some of these ranges removed from the test? It looks like it was giving you the range of the whole module itself.

Reply:
It's removing ranges of anon top level modules.

Reply:
I see, yes that makes sense. That's just an inner module.

Reply:
Oh that's nice